v0/destinations/posthog/data/PHScreenConfig.json

```diff
@@ -0,0 +1,23 @@
+[
+  { "destKey": "properties", "sourceKeys": "properties", "required": false },
+  {
+    "destKey": "timestamp",
+    "sourceKeys": ["originalTimestamp", "timestamp"],
+    "required": false
+  },
+  { "destKey": "category", "sourceKeys": "category", "required": false },
+  {
+    "destKey": "distinct_id",
+    "sourceKeys": [
+      "userId",
+      "traits.userId",
+      "traits.id",
+      "context.traits.userId",
+      "context.traits.id",
+      "anonymousId"
+    ],
+    "required": true
+  },
+  { "destKey": "name", "sourceKeys": "name", "required": false },
```

Comment:
Should we give precedence to `properties.name` as well?

Reply:
It is not required for PostHog. As we map the whole Rudder `properties` to PostHog `properties`

Reply:
> Should we give precedence to `properties.name` as well?

It is not required for PostHog. As we map the whole Rudder properties to PostHog properties
@arnab-p

Reply:
No Changes made.